nemo/collections/nlp/metrics/qa_metrics.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright 2019 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import string
+from collections import Counter
+
+
+class QAMetrics(object):
+    @classmethod
```

Comment:
I noticed you are using classmethods here - you should have a look at staticmethod vs classmethod and I generally recommend staticmethod if you're not accessing class variables.